org.ekstep.lessonbrowser-1.4/editor/lessonBrowserApp.js

```diff
@@ -422,7 +422,8 @@ angular.module('org.ekstep.lessonbrowserapp', ['angular-inview', 'luegg.directiv
                         var contents = [];
                         $scope.mainTemplate = 'facetsItemView';
                         ctrl.applyAllJquery();
-                        if (ctrl.facetsResponse.result.response.sections) {
+                        var contentCount = _.findIndex(ctrl.facetsResponse.result.response.sections, function(o) { return o.contents });
+                        if (ctrl.facetsResponse.result.response.sections && contentCount > 0) {
```

Comment:
How it works if the contents found in the `0th` index of section? You are basically doing `_findIndex` right?

`_.findIndex` always return position right?